### PR TITLE
Add base to be include when fetching plugin manifest and entry

### PR DESCRIPTION
**Pull request by karelhala**
### UI trying to fetch plugins from non-beta env

When on any beta environment the UI is trying to fetch dynamic plugin from root route. This is not correct as we have to support beta environment first. This PR changes such thing by prepending fetching manifests and plugin entries with calculated base from chrome API.

### CODE REVIEW

**Comment by karelhala** [comment]
ping @andrewballantyne

File: frontend/src/Navigation.ts [hunk 1]
```diff
@@ -33,7 +33,8 @@ const isNavItem = (extension: Extension): extension is NavExtension => {
 const getAllExtensions: GetAllExtensions = async () => {
   const results: PromiseSettledResult<Extension[]>[] = await Promise.allSettled(
     activePlugins.flatMap(async ({ name: pluginName, pathPrefix = '/api/plugins' }: EnabledPlugin) => {
-      const url = `${pathPrefix}/${pluginName}/plugin-manifest.json`;
+      // TODO: once we are hook use chrome to calculate root
+      const url = `./${pathPrefix}/${pluginName}/plugin-manifest.json`;
```
Comment (andrewballantyne) [inline frontend/src/Navigation.ts:37]: Will this work? 🤔 

Seems odd why we would use a base path at all if this works.

Comment (karelhala) [reply frontend/src/Navigation.ts:37]: It works on deployed applications, locally basePath is not correct for some reason, that's why we are using the chrome's hook to get the proper env (including /beta if on beta env).

Comment (andrewballantyne) [reply frontend/src/Navigation.ts:37]: Then isn't this a moot change?

Comment (karelhala) [reply frontend/src/Navigation.ts:37]: Yeah! You are correct, since we are in hook I can use chrome's hook to calculate the base.

**Review by andrewballantyne — APPROVED** [review]

**Comment by andrewballantyne** [comment]
@karelhala I think we are fully merged on the CI of things... I believe that was the only reason this was held up. I've approved, un "WIP" it when you want to merge.

**Pull request merged**